src/icesat2_tracks/ICEsat2_SI_tools/iotools.py

```diff
@@ -193,7 +183,7 @@ def set_ATL03_trackname(self):
     def set_ATL10_trackname(self):
         block1 = (self.YY, self.MM, self.DD)
         block1b = (self.HH, self.MN, self.SS)
-        block2 = (self.TRK, self.CYC, "01")  # granule is alwasy '01' for ATL10
+        block2 = (self.TRK, self.CYC, "01")
```

Comment:
This feels like a comment to keep:
```suggestion
        block2 = (self.TRK, self.CYC, "01")  # granule is always '01' for ATL10
```

Reply:
comment added